src/vs/workbench/contrib/chat/browser/actions/chatActions.ts

```diff
@@ -32,8 +32,47 @@ import { LifecyclePhase } from 'vs/workbench/services/lifecycle/common/lifecycle
 import { AccessibilityHelpAction } from 'vs/workbench/contrib/accessibility/browser/accessibleViewActions';
 
 export const CHAT_CATEGORY = { value: localize('chat.category', "Chat"), original: 'Chat' };
+export const CHAT_OPEN_ACTION_ID = 'workbench.action.chat.open';
+
+class QuickChatGlobalAction extends Action2 {
+	constructor() {
+		super({
+			id: CHAT_OPEN_ACTION_ID,
+			title: { value: localize('quickChat', "Quick Chat"), original: 'Quick Chat' },
+			precondition: CONTEXT_PROVIDER_EXISTS,
+			icon: Codicon.commentDiscussion,
+			f1: false,
+			category: CHAT_CATEGORY,
+			keybinding: {
+				weight: KeybindingWeight.WorkbenchContrib,
+				primary: KeyMod.CtrlCmd | KeyMod.Alt | KeyCode.KeyI,
```

Comment:
You can remove the keybinding from the extension if you haven't done that already

Reply:
https://github.com/microsoft/vscode-copilot/pull/1355 boom